Dockerfile

```
@@ -0,0 +1,21 @@
+FROM php:7.2-fpm
+
+# Install PHP extensions
+RUN docker-php-ext-install pdo_mysql && docker-php-ext-install calendar
+
+# Install Composer
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+
+# Install Git
+RUN apt-get update && apt-get install -y git
```

```suggestion
RUN apt-get update \
  && apt-get install -y git \
  && rm -rf /var/lib/apt/lists/*
```

Those apt lists are quite some MBs so make sure to delete them.